server/src/controllers/course.js

```diff
@@ -0,0 +1,25 @@
+const { getcourseByCatId } = require('../database/queries');
+
+exports.getCoursesByCatId = async (req, res, next) => {
+  try {
+    const { categoryId } = req.params;
+    if (categoryId > 0) {
+      const { rows } = await getcourseByCatId(categoryId);
+      if (rows.length !== 0) {
+        res.json({ statusCode: 200, data: rows });
```

Comment:
**here, it's enough to send `rows` only because by default  `statusCode` will be 200 so, there is no need to add it.**
```suggestion
        res.json(rows );
```